AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -857,7 +857,7 @@ - (void)layoutSubviews
   _nextLayoutInvalidationStyle = ASCollectionViewInvalidationStyleNone;
   switch (invalidationStyle) {
     case ASCollectionViewInvalidationStyleWithAnimation:
-      if (!_superPerformingBatchUpdates) {
+      if (0 == _superBatchUpdateCount) {
```

Comment:
FWIW, I'm pretty sure the compiler catches a single = for you now. 
<img width="263" alt="screen shot 2016-07-15 at 4 19 30 pm" src="https://cloud.githubusercontent.com/assets/285809/16890907/ee09097a-4aa7-11e6-9f4b-00148c893379.png">